rules/windows/execution_wpad_exploitation.toml

```diff
@@ -0,0 +1,51 @@
+[metadata]
+creation_date = "2020/09/02"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/09/02"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies probable exploitation of the Web Proxy Auto-Discovery Protocol (WPAD) service. Attackers who have access to
+the local network or upstream DNS traffic can inject malicious JavaScript to the WPAD service which can lead to a full
+system compromise.
+"""
+index = ["logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License"
+name = "WPAD Service Exploit"
+risk_score = 21
+rule_id = "ec328da1-d5df-482b-866c-4a435692b1f3"
+severity = "high"
+tags = ["from-eql-rules"]
+type = "eql"
+
+query = '''
+sequence with maxspan=5s
+  [process where event.type in ("start", "process_started") and process.name == "svchost.exe" and
+     user.domain == "NT AUTHORITY" and user.name == "LOCAL SERVICE"] by process.entity_id, process.pid, process.name
```

Comment:
use `user.sid` if possible. `user.domain` of "NT AUTHORITY" is locale specific, so this would only work in English locales.

Reply:
also, I don't remember this rule ever firing in production, so I think it's okay to nix altogether

Reply:
I don't see `user.sid` in ECS nor either source.

If it doesn't test well, I can drop to `development`

Reply:
I will add this as a comment to preserve it too

Reply:
```suggestion
     user.domain == "NT AUTHORITY" and user.name == "LOCAL SERVICE"] by process.entity_id
```